src/Libraries/DSOffice/Excel.cs

```diff
@@ -350,6 +350,14 @@ private static object[][] ConvertToJaggedArray(object[,] input)
                     cols = Math.Max(cols, input[i].GetUpperBound(0) + 1);
             }
 
+            // if the input data is an empty list or a list of nested empty lists
+            // return an empty cell
+            if(rows == 0 || cols == 0)
```

Comment:
Good point but I discussed this with @monikaprabhu. We decided that while this might be consistent for lists that have nested empty lists and non-empty lists in them, it might not be necessary for lists that contained nothing but empty lists.